src/lib.rs

```diff
@@ -190,8 +202,8 @@ impl EguiClipboard {
 
     #[cfg(target_arch = "wasm32")]
     #[allow(clippy::unnecessary_wraps)]
-    fn get_contents_impl(&self) -> Option<String> {
-        Some(self.clipboard.clone())
+    fn get_contents_impl(&mut self) -> Option<String> {
+        self.clipboard.try_read_clipboard_event()
     }
```

Comment:
That's a bit funny how reading from clipboard requires `&mut self`, but writing to it doesn't 😄

Reply:
yeah, I guess we could hide this by caching the web event, but that would require to:
- insert a system retrieving the web event (`try_recv`), store the result
- get that result here with a non mut access

I'm not sure about its benefits.

Reply:
Yeah, I'm fine with keeping it `&mut self`, it was just an observation from my end :)